packages/rspack/src/index.ts

```diff
@@ -63,4 +63,12 @@ export const wasm = { EnableWasmLoadingPlugin };
 import { LimitChunkCountPlugin } from "./builtin-plugin";
 export const optimize = { LimitChunkCountPlugin };
 
+export const sources = require("webpack-sources");
+
+export const util = {
+	get serialization() {
+		return require("./util/serialization");
```

Comment:
Don't export this for now, we haven't implement serialization related stuff in Rspack